crd-docs/crd/kiali.io_kialis.yaml

```diff
@@ -1022,6 +1022,24 @@ spec:
                             secret_name:
                               description: "The name of the secret that contains the credentials necessary to connect to the remote cluster. This secret must exist in the Kiali deployment namespace."
                               type: string
+                      kiali_urls:
+                        description: "A map between cluster name, instance name and namespace to a Kiali URL. Will be used showing the Mesh page's Kiali URLs. The Kiali service's 'kiali.io/external-url' annotation will be overridden when this property is set."
+                        type: array
+                        items:
+                          type: object
+                          properties:
+                            cluster_name:
+                              description: "The name of the cluster."
+                              type: string
+                            instance_name:
+                              description: "The instance name of this Kiali installation."
```

Comment:
when you say "instance name of this Kiali" , is this referencing the "deployment.instance_name" of the Kiali install? If so, you might want to explicitly mention that here to make it clear to the user this value needs to match that setting for that Kiali installation.

Reply:
agree, fixed